metpy/calc/tests/test_basic.py

```diff
@@ -234,7 +234,7 @@ def test_heat_index_invalid():
     rh = np.array([40, 39, 2, 70, 50, 39]) * units.percent
```

Comment:
Can you add an additional point that should be masked here?

Reply:
I am unsure if I follow this, presently 2 of the 6 values are masked.  Could you kindly elaborate on this request or provide what additional point you have in mind?

Reply:
Um... :man_shrugging: 🐑 Apparently I need to step away from the screen. Nevermind.